Simplenote/src/main/java/com/automattic/simplenote/AboutActivity.java

```diff
@@ -13,7 +13,6 @@
 public class AboutActivity extends AppCompatActivity {
```

Comment:
We can remove the `import com.automattic.simplenote.utils.ThemeUtils;` statement since it is no longer used.